addons/website/static/src/xml/website.editor.xml

```diff
@@ -37,17 +37,29 @@
         <div class="form-row justify-content-between o_options_container"/>
     </div>
     <t t-name="website.theme_customize_modal_option">
-        <div t-attf-class="o_theme_customize_option my-1 flex-grow-0 #{font ? 'o_theme_customize_option_font_' + font : ''} #{widget ? 'o_theme_customize_with_widget' : ''}">
+        <a t-if="font or label" href="#" t-attf-class="dropdown-item p-0 o_theme_customize_option #{font ? 'o_theme_customize_option_font_' + font : ''}">
+           <label class="justify-content-start" t-att-data-font-id="font" t-att-data-label="label">
+               <i class="fa fa-check mr-1"/>
+               <input t-att-id="id" t-att-name="name" type="radio" t-att-checked="checked ? 'checked' : undefined"/>
+               <span><t t-esc="label"/></span>
+           </label>
+       </a>
```

Comment:
adding the option in a dropdown item should be something handled on the JS side, and only when using a selection tag (not related to font or label)